src/components/ebay-table/component.ts

```diff
@@ -105,4 +126,21 @@ export default class EbayTable extends Marko.Component<Input, State> {
             selected: this.state.selected,
         });
     }
+
+    sortColumn(name: TableColRowName) {
+        const sort = this.state.sorted[name];
+        if (sort) {
```

Comment:
Might be cleaner to have a map in this case
```
const sortTo = {
   "asc": "desc",
   "desc": "none",
   "none": "asc"
}

this.state.sorted[name] = sortTo[sort || "none"];
```
Or we can do switch and case here.